scrapyd_k8s/config_loader.py

```diff
@@ -0,0 +1,3 @@
+from .config import Config
+
+config = Config()
```

Comment:
I'm a little weary to do it like this: can we make an instance of it in the main file, and pass that on where needed?

Reply:
If resides in the api.py then I have circular imports, that is why I decided that it should be separate to not cause conflicts. I can also create the config instance in the file with the Config class and import it from there

Reply:
So, we cannot pass the config from api.py to logging_config.py to __main__.py, we can of course move the logging configuring and related things to the api.py but then we are mixing very different things in one file and end up with a bulk of coupled code. Are you ok with keeping the config instance as a separate logic?